### ISSUE TITLE
Bug: database prefix on run php artisan voyager:install

### ISSUE THREAD

**Issue by enniosousa** [thedevdojo/voyager#4541]
## Version information
 - Laravel: v^6.2
 - Voyager: v^1.3
 - PHP: 7.3
 - Database: SQL Server

## Description
the bug occurred when I ran the `php artisan voyager:install` command. I'm using database prefix, and it is cause of bug

## Steps To Reproduce
```shell
root@srvapiseduc:/var/www/project# php artisan voyager:install
Publishing the Voyager assets, database, and config files
Copied Directory [/vendor/tcg/voyager/publishable/database/seeds] To [/database/seeds]
Publishing complete.
Publishing complete.
Unable to locate publishable resources.
Publishing complete.
Migrating the database tables into your application
Migration table created successfully.
Migrating: 2014_10_12_000000_create_users_table
Migrated:  2014_10_12_000000_create_users_table (0.01 seconds)
Migrating: 2014_10_12_100000_create_password_resets_table
Migrated:  2014_10_12_100000_create_password_resets_table (0.01 seconds)
Migrating: 2016_01_01_000000_add_voyager_user_fields
Migrated:  2016_01_01_000000_add_voyager_user_fields (0.02 seconds)
Migrating: 2016_01_01_000000_create_data_types_table
Migrated:  2016_01_01_000000_create_data_types_table (0.04 seconds)
Migrating: 2016_05_19_173453_create_menu_table
Migrated:  2016_05_19_173453_create_menu_table (0.01 seconds)
Migrating: 2016_10_21_190000_create_roles_table
Migrated:  2016_10_21_190000_create_roles_table (0.01 seconds)
Migrating: 2016_10_21_190000_create_settings_table
Migrated:  2016_10_21_190000_create_settings_table (0.01 seconds)
Migrating: 2016_11_30_135954_create_permission_table
Migrated:  2016_11_30_135954_create_permission_table (0.01 seconds)
Migrating: 2016_11_30_141208_create_permission_role_table
Migrated:  2016_11_30_141208_create_permission_role_table (0.05 seconds)
Migrating: 2016_12_26_201236_data_types__add__server_side
Migrated:  2016_12_26_201236_data_types__add__server_side (0 seconds)
Migrating: 2017_01_13_000000_add_route_to_menu_items_table
Migrated:  2017_01_13_000000_add_route_to_menu_items_table (0.01 seconds)
Migrating: 2017_01_14_005015_create_translations_table
Migrated:  2017_01_14_005015_create_translations_table (0.01 seconds)
Migrating: 2017_01_15_000000_make_table_name_nullable_in_permissions_table
Migrated:  2017_01_15_000000_make_table_name_nullable_in_permissions_table (0.17 seconds)
Migrating: 2017_03_06_000000_add_controller_to_data_types_table
Migrated:  2017_03_06_000000_add_controller_to_data_types_table (0 seconds)
Migrating: 2017_04_21_000000_add_order_to_data_rows_table
Migrated:  2017_04_21_000000_add_order_to_data_rows_table (0.01 seconds)
Migrating: 2017_07_05_210000_add_policyname_to_data_types_table
Migrated:  2017_07_05_210000_add_policyname_to_data_types_table (0 seconds)
Migrating: 2017_08_05_000000_add_group_to_settings_table
Migrated:  2017_08_05_000000_add_group_to_settings_table (0.01 seconds)
Migrating: 2017_11_26_013050_add_user_role_relationship
Migrated:  2017_11_26_013050_add_user_role_relationship (0.14 seconds)
Migrating: 2017_11_26_015000_create_user_roles_table

   Doctrine\DBAL\Schema\SchemaException  : There is no column with name 'id' on table 'users'.

  at /var/www/project/vendor/doctrine/dbal/lib/Doctrine/DBAL/Schema/SchemaException.php:82
    78|      * @return \Doctrine\DBAL\Schema\SchemaException
    79|      */
    80|     public static function columnDoesNotExist($columnName, $table)
    81|     {
  > 82|         return new self(
    83|             sprintf("There is no column with name '%s' on table '%s'.", $columnName, $table),
    84|             self::COLUMN_DOESNT_EXIST
    85|         );
    86|     }

  Exception trace:

  1   Doctrine\DBAL\Schema\SchemaException::columnDoesNotExist("id", "users")
      /var/www/project/vendor/doctrine/dbal/lib/Doctrine/DBAL/Schema/Table.php:660
```

**Comment by MrCrayon** [comment]
Can you try to add ```DB::getTablePrefix()``` in that migration before running install?
```php
$type = DB::connection()->getDoctrineColumn(DB::getTablePrefix().'users', 'id')->getType()->getName();
```

Edit:
I saw you already successfully tried that.

**Comment by trunganhvu** [comment]
i have a bug same this post. i tried but not ok


/opt/lampp/htdocs/myfilm1$ php artisan voyager:install --with-dummy
Publishing the Voyager assets, database, and config files
Copied Directory [/vendor/tcg/voyager/publishable/database/seeds] To [/database/seeds]
Publishing complete.
Publishing complete.
<b>Unable to locate publishable resources.</b> <- error
Publishing complete.
Migrating the database tables into your application
Migrating: 2016_01_01_000000_add_voyager_user_fields
Migrated:  2016_01_01_000000_add_voyager_user_fields (0.63 seconds)
Migrating: 2016_01_01_000000_create_data_types_table
Migrated:  2016_01_01_000000_create_data_types_table (2.4 seconds)
Migrating: 2016_05_19_173453_create_menu_table
Migrated:  2016_05_19_173453_create_menu_table (2.32 seconds)
Migrating: 2016_10_21_190000_create_roles_table
Migrated:  2016_10_21_190000_create_roles_table (0.77 seconds)
Migrating: 2016_10_21_190000_create_settings_table
Migrated:  2016_10_21_190000_create_settings_table (0.48 seconds)
Migrating: 2016_11_30_135954_create_permission_table
Migrated:  2016_11_30_135954_create_permission_table (0.79 seconds)
Migrating: 2016_11_30_141208_create_permission_role_table
Migrated:  2016_11_30_141208_create_permission_role_table (5.11 seconds)
Migrating: 2016_12_26_201236_data_types__add__server_side
Migrated:  2016_12_26_201236_data_types__add__server_side (0.09 seconds)
Migrating: 2017_01_13_000000_add_route_to_menu_items_table
Migrated:  2017_01_13_000000_add_route_to_menu_items_table (0.09 seconds)
Migrating: 2017_01_14_005015_create_translations_table
Migrated:  2017_01_14_005015_create_translations_table (0.47 seconds)
Migrating: 2017_01_15_000000_make_table_name_nullable_in_permissions_table
Migrated:  2017_01_15_000000_make_table_name_nullable_in_permissions_table (0.62 seconds)
Migrating: 2017_03_06_000000_add_controller_to_data_types_table
Migrated:  2017_03_06_000000_add_controller_to_data_types_table (0.09 seconds)
Migrating: 2017_04_21_000000_add_order_to_data_rows_table
Migrated:  2017_04_21_000000_add_order_to_data_rows_table (0.08 seconds)
Migrating: 2017_07_05_210000_add_policyname_to_data_types_table
Migrated:  2017_07_05_210000_add_policyname_to_data_types_table (0.09 seconds)
Migrating: 2017_08_05_000000_add_group_to_settings_table
Migrated:  2017_08_05_000000_add_group_to_settings_table (0.54 seconds)
Migrating: 2017_11_26_013050_add_user_role_relationship
Migrated:  2017_11_26_013050_add_user_role_relationship (2.81 seconds)
Migrating: 2017_11_26_015000_create_user_roles_table
Migrated:  2017_11_26_015000_create_user_roles_table (3.81 seconds)
Migrating: 2018_03_11_000000_add_user_settings
Migrated:  2018_03_11_000000_add_user_settings (0.11 seconds)
Migrating: 2018_03_14_000000_add_details_to_data_types_table
Migrated:  2018_03_14_000000_add_details_to_data_types_table (0.22 seconds)
Migrating: 2018_03_16_000000_make_settings_value_nullable
Migrated:  2018_03_16_000000_make_settings_value_nullable (0.67 seconds)
Attempting to set Voyager User model as parent to App\User
Dumping the autoloaded files and reloading all new files
Adding Voyager routes to routes/web.php
Seeding data into the database
Publishing dummy content
Copied Directory [/vendor/tcg/voyager/publishable/database/dummy_seeds] To [/database/seeds]
Publishing complete.
Copied Directory [/vendor/tcg/voyager/publishable/dummy_content] To [/storage/app/public]
Publishing complete.
Copied File [/vendor/tcg/voyager/publishable/config/voyager_dummy.php] To [/config/voyager.php]
Publishing complete.
Copied Directory [/vendor/tcg/voyager/publishable/database/migrations] To [/database/migrations]
Publishing complete.
Publishing complete.
Migrating dummy tables
Migrating: 2016_01_01_000000_create_pages_table
Migrated:  2016_01_01_000000_create_pages_table (0.71 seconds)
Migrating: 2016_01_01_000000_create_posts_table
Migrated:  2016_01_01_000000_create_posts_table (0.71 seconds)
Migrating: 2016_02_15_204651_create_categories_table
Migrated:  2016_02_15_204651_create_categories_table (1.94 seconds)
Migrating: 2017_04_11_000000_alter_post_nullable_fields_table
Migrated:  2017_04_11_000000_alter_post_nullable_fields_table (0.68 seconds)
Seeding dummy data
Setting up the hooks
Copied File [/vendor/larapack/hooks/publishable/config/hooks.php] To [/config/hooks.php]
Publishing complete.
Publishing complete.
Hooks are now ready to use! Go ahead and try to "php artisan hook:install test-hook"
Adding the storage symlink to your public folder
The [public/storage] directory has been linked.
Successfully installed Voyager! Enjoy

**Comment by MrCrayon** [comment]
@trunganhvu that's not the same error, there is already another bug report about that with suggested fix #4555
Plus as you can see 

> Successfully installed Voyager! Enjoy

It doesn't affect Voyager installation, it only matters if you want to change Image standard settings because you'll have to publish config manually.

**Comment by trunganhvu** [comment]
> @trunganhvu that's not the same error, there is already another bug report about that with suggested fix #4555
> Plus as you can see
> 
> > Successfully installed Voyager! Enjoy
> 
> It doesn't affect Voyager installation, it only matters if you want to change Image standard settings because you'll have to publish config manually.

i am newbie. i cant login with admin@admin.com/password the notification : These credentials do not match our records. 
Do you know it?

**Comment by MrCrayon** [comment]
This is not related to this issue.
Please check https://voyager-docs.devdojo.com/ if after reading there you think you found a bug you can open a new issue.
If you need help related to Voyager you can ask in Voyager Slack group.
